modules/instrain/profile/main.nf

```diff
@@ -0,0 +1,46 @@
+process INSTRAIN_PROFILE {
+    tag "$meta.id"
+    label 'process_high'
+    label 'error_retry'
+    label 'process_long'
+
+    conda (params.enable_conda ? "bioconda::instrain=1.6.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/instrain%3A1.6.1--pyhdfd78af_0':
+        'quay.io/biocontainers/instrain:1.6.1--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(bam)
+    path genome_fasta
+    path genes_fasta
+    path stb_file
+
+    output:
+    tuple val(meta), path("*.IS") , emit: profile
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def genes_args = genes_fasta != [] ? "-g ${genes_fasta}": ''
```

Comment:
```suggestion
    def genes_args = genes_fasta ? "-g ${genes_fasta}": ''
```

By default just invoking the name of the variable in a condition results in a boolean output, you don't need to define the condition